BadHabbit/BadHabbit/UserStories/SingUpScreen/SignUpScreenAssembly.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  SignUpScreenAssembly.swift
+//  BadHabbit
+//
+//  Created by Viktoria Misiulia on 27/07/2023.
+//
+
+import UIKit
+
+class SignUpScreenAssembly: StoryAssembly {
+    private let nextScreen: StoryAssembly
+    private let globalCoordinator: IGlobalCoordinator
+    
+    init(nextScreen: StoryAssembly, globalCoordinator: IGlobalCoordinator) {
+        self.nextScreen = nextScreen
+        self.globalCoordinator = globalCoordinator
+    }
+    
+    func assembleStory() -> UIViewController {
+        let router = SignUpScreenRouter(nextScreen: nextScreen,
```

Comment:
Please, don't allign several parameters like that. Start first parameter with new line, other parameters aligned with first one

Reply:
Alignment was changed